.github/workflows/build-test.yml

```diff
@@ -158,7 +176,7 @@ jobs:
           export PATH=../deps/nats-server:../deps/nats-streaming-server:$PATH
           export NATS_TEST_SERVER_VERSION="$(nats-server -v)"
           flags=""
-          ctest --timeout 60 --output-on-failure --repeat-until-fail ${{ inputs.repeat }} 2>&1 | tee /tmp/out.txt
+          ctest $VVFLAG --timeout 60 --output-on-failure --repeat-until-fail ${{ inputs.repeat }} 2>&1 | tee /tmp/out.txt
```

Comment:
This is strange. At least on travis, the test would fail if there is an error found by `stanitize-(address/test)` without the need to have verbose test output. Why is it different with Github-Actions? I mean the `ctest` should fail (return non zero and therefore make the run fail). Is it that the sanitize output is not present in GA while it is in Travis, and that makes acting on the failure impossible for us dev?

Reply:
Ok, I think I understand. I have just compiled on my Ubuntu VM with stanitize-thread and I see that the test passes but if I enable logging (-V) then I see the warning. So it was not considered an error and did not fail the test. So I guess yes, it is worth having verbose test output to catch warnings too.

Reply:
I found a better solution, using TSAN_OPTIONS, will push to this PR shortly.

Reply:
Good. I would recommend removing the ifdef for the force reconnect test to verify that it works, and then you could replace the socket close with shutdown to verify that it works.